src/core.js

```diff
@@ -51,11 +51,12 @@ function createModule( name, testEnvironment, modifiers ) {
 		childModules: [],
 		suiteReport: new SuiteReport( name, parentSuite ),
 
-		// Pass along `skip` property from parent module, in case there is one,
-		// to childs. And use own otherwise.
+		// Pass along `skip` and `todo` properties from parent module, in case
+		// there is one, to childs. And use own otherwise.
 		// This property will be used to mark own tests and tests of child suites
-		// as `skipped`.
-		skip: parentModule !== null && parentModule.skip || modifiers.skip
+		// as either `skipped` or `todo`.
+		skip: parentModule !== null && parentModule.skip || modifiers.skip,
+		todo: parentModule !== null && parentModule.todo || modifiers.todo
```

Comment:
Yes, you are right, but will have no side effects AFAIK. Because the only place I use those properties are in file `src/test.js#L46-L52`. And in case they are both `true`, only `skip` will then be used.

Anyway, I updated it.

Thanks!